docker/neo4j/README.md

```diff
@@ -35,24 +37,33 @@ gcloud auth configure-docker
 ```shell
 docker -- pull marketplace.gcr.io/google/neo4j4
 ```
-## Table of Contents
+Dockerfile for this image can be found [here](https://github.com/GoogleCloudPlatform/click-to-deploy/tree/master/docker/neo4j/4/debian9/4.3).
+
+# <a name="table-of-contents"></a>Table of Contents
 
  [Using Docker](#using-docker)
-  * [Run a server](#run-a-activemq-server-docker)
-    * [Running Neo4j Service](#Running-Neo4j-service)
-  * [Use a persistent data volume docker](#Use-a-persistent-data-volume)
-  * [Variables, paths and ports](#Variables)
+  * [Run a server](#run-a-Neo4j-server-docker)
+    * [Running Neo4j Service](#running-Neo4j-service)
+  * [Use a persistent data volume docker](#use-a-persistent-data-volume)
+  * [References](#references)
+   * [Ports](#references-ports)
```

Comment:
Here it's missing one more space so you can nest Ports, Env Vars and Volumes one lever deeper